raster-foundry/raster-foundry
# Fix social login / refresh token creation

## Overview
Update auth0-lock to 10.24.0
Use redirecting logins instead of popups
Consolidate login page to single module file

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

## Testing Instructions

 * Make sure auth0 is in oidc compliant mode
* Verify that social logins and the api refresh tokens work correctly, and use redirection instead of popups

Closes #2691
